### ISSUE TITLE
10$: Improve diagnostics for var\nvar\nvar

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#233]
quick-lint-js emits confusing diagnostics for the following code:

```js
var
var
var
```
```
hello.js:1:1: error: let with no bindings [E024]
hello.js:3:1: error: cannot declare variable named keyword 'var' [E124]
```

quick-lint-js should report E024 for each line.

**Comment by LeeWannacott** [comment]
I claim this for-hire task. I expect payment after I complete this task. I will email the quick-lint-js team if I am assigned this task.

**Comment by LeeWannacott** [comment]
Question :thinking: : Does each line mean? 
```
hello.js:1:1: error: var with no bindings [E024]
hello.js:2:1: error: var with no bindings [E024]
hello.js:3:1: error: var with no bindings [E024]
```
Or:
```
hello.js:1:1: error: var with no bindings [E024]
hello.js:3:1: error: var with no bindings [E024]
```

**Comment by strager** [comment]
> Question 🤔 : Does `each line` mean?
> 
> ```
> hello.js:1:1: error: var with no bindings [E024]
> hello.js:2:1: error: var with no bindings [E024]
> hello.js:3:1: error: var with no bindings [E024]
> ```
> 
> Or:
> 
> ```
> hello.js:1:1: error: var with no bindings [E024]
> hello.js:3:1: error: var with no bindings [E024]
> ```
> 
> Note: After the arduous issue of #234 it now outputs error: var, not let.

The former: 3 errors, one per `var`.

**Comment by strager** [comment]
I think improving this isn't worth the engineering effort.
